apps/api/src/app/layouts/usecases/create-layout/create-layout.use-case.ts

```diff
@@ -39,10 +39,10 @@ export class CreateLayoutUseCase {
         userId: dto._creatorId,
       });
       await this.setDefaultLayout.execute(setDefaultLayoutCommand);
+    } else {
+      await this.createChange(command, dto._id);
```

Comment:
I wonder what would happen if there is an update/create of layout(name,content) + setting to default. Would it create 2 changes? One for default layout change and one for layout change? If so, are we sure they are also dependent on promote?

Reply:
In that case, It will create a single(`1`) change under `DefaultLayout`,
This way it maintains the dependency between other layout changes too.